client/components/mma/identity/form/FormField.tsx

```diff
@@ -47,6 +47,7 @@ const FormField = <T,>(props: FormFieldProps<T>) => {
 			{label}
 			{field}
 			{error ? <p>{error}</p> : null}
+			data-qm-masking="blocklist"
```

Comment:
This is inside the `label` element rather than being applied as an attribute so will get rendered out as text.

Reply:
Where would you put it? Should it be in a new div?

Reply:
It could go on the `label` element that's wrapping everything. Alternatively, maybe adding a `span` just around the input field so the label doesn't get masked as well. It's tricky to know without testing as adding extra elements could cause layout issues depending on the styles.